Samples/Uno/LibVLCSharp.Uno.Sample.Shared/LibVLCSharp.Uno.Sample.Shared.shproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hmm the Uno people seem to like their shared projects :D

Reply:
Yes, I don't like too but it's the default template when you create an Uno project, I preferred to let it like that.

Reply:
Sure, just making an observation :+1: